### PR TITLE
Add a basic pytest

### PR DESCRIPTION
**Pull request by madwort**
* Test the construction of the jacktrip command line
* Tiny refactor to make it easier to test
* https://docs.pytest.org/en/latest/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,17 @@
 Python scripts for AUTONOMOUS_NOISE_UNIT interface.
 
 more to come... for now [see here](https://github.com/noiseorchestra/noise-audio-web).
+
+## Tests
+
+### Install
+
+```bash
+$ python3 -m pip install pytest
```
Comment (madwort) [inline README.md:11]: we should use a package manager tho....

Comment (madwort) [reply README.md:11]: https://pip.pypa.io/en/latest/user_guide/#requirements-files

Comment (madwort) [reply README.md:11]: EDIT: IT'S ALREADY THERE I JUST IGNORED IT!

File: requirements.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ psutil==5.7.2
 pycparser==2.20
 pyftdi==0.51.2
 pyserial==3.4
+pytest==5.4.3
```
Comment (madwort) [inline requirements.txt:11]: HERE'S THE BADGER

**Comment by noiseorchestra** [comment]
Thanks for this Tom! I'm closing because the relevant code has now changed, but testing is in the process of being implemented :+1:

**Pull request closed without merging**